app/controllers/obituaries/covid_controller.rb

```diff
@@ -1,5 +1,7 @@
 class Obituaries::CovidController < ApplicationController
   def index
-    @obituaries = Obituary.where(covid: true) 
+    @obituaries_api = ObituarySearch.new.search_covid_obituaries
+    @obituaries = Obituary.where(covid: true)
```

Comment:
I'm wondering if we want to add a sort to this, but that is probably better left to a future iteration!